db/error.js

```diff
@@ -0,0 +1,16 @@
+exports.handlePsqlErrors = (err,req,res,next) => {
+    if (err.code === "22P02") {
+        res.status(400).send({msg:"Bad request"})
+    } else next (err);
+
+};
+
+exports.handleCustomErrors = (err,req,res,next) => {
+    if(err.msg) {
+        res.status(err.status).send({msg:err.msg});
+    } else next (err);
+};
+
+exports.handleServerErrors = (err,req,res,next) => {
+   res.status(500).send({msg:"Server broken..."});
+};
```

Comment:
It's fine to have a 500 error handler at this stage, but do we need the other 2 error handlers? We should only be adding error handling when the tests you have written requires it. Please remove anything that's unnecessary. 👍

Reply:
I have removed  psql and custom error handler and keep server error handler.